app/src/androidTest/java/com/github/HumanLearning2021/HumanLearningApp/DisplayDatasetActivityTest.kt

```diff
@@ -80,11 +82,12 @@ class DisplayDatasetActivityTest {
         }
     }
 
-    @Test
-    fun modifyingDatasetNameWorks() {
-        onView(withId(R.id.display_dataset_name)).perform((typeText("Dataset Name\n")))
-        onView(withId(R.id.display_dataset_name)).check(matches(withText(containsString("Dataset Name"))))
-    }
+//    @Test
+//    fun modifyingDatasetNameWorks() {
+//        val newName = "new dataset name"
+//        onView(withId(R.id.display_dataset_name)).perform((typeText("$newName\n")))
+//        onView(withId(R.id.display_dataset_name)).check(matches(withText(containsString(newName))))
+//    }

```

Comment:
Yes, because the test depend on each other this one breaks all the others. I tried fixing it by using Dependecy Injection but encountered some issues. After talking with Nicolas we decided to not loose any more time on it and fix it on next sprint.